service/resources/helpers/helpers.py

```diff
@@ -50,6 +50,11 @@ def get_emails(emails, email_type):
         email_list = []
         counter = 0
         for email in emails:
+            if email_type != 'to' and \
```

Comment:
Tried to do the minimal check here, so excluded the `to` fields... if you send those improperly, it's still going to bail like previously.. but it should happen less since the `to` field seems kinda like a no-brainer?